### ISSUE TITLE
"let g:Tex_GotoError=0" is not working

### ISSUE THREAD

**Issue by jiayouwyhit** [vim-latex/vim-latex#96]
I find that there is a conflict between the setting of "g:Tex_MultipleCompileFormats" and the setting of "Tex_GotoError".

I want to disable the automated jumping to error page by doing "let g:Tex_GotoError=0". But it does not work, since I have another setting: "let g:Tex_MultipleCompileFormats='pdf' ". If I change  Tex_MultipleCompileFormats to another value, then "let g:Tex_GotoError=0" can work.
It is exactly same to this link: [gtex_gotoerror0-is-not-working-in-vim-latex](https://huangkan.wordpress.com/2014/02/22/how-to-solve-gtex_gotoerror0-is-not-working-in-vim-latex/)

Then, how should I solve this issue?

**Comment by jiayouwyhit** [comment]
Btw, I am currently using the default settings of spf13 to config my vim settings. Not quite sure if there is anything changed by spf13.

**Comment by gerw** [comment]
I cannot reproduce this behavior. In principle, `let g:Tex_GotoError=0` and `let g:Tex_MultipleCompileFormats='pdf'` are not related in the source code.

Please enable debugging
````
let g:Tex_Debug = 1
let g:Tex_DebugLog = "vim-latex-suite.log"
````
and post the results (with and without the error occurring).

**Comment by jiayouwyhit** [comment]
Thanks a lot, @gerw . I followed your instructions and go the following results as shown in the vim-latex-suite.log file. The issue still exists.
`
proj : texproject.vim: catching LatexSuiteFileType event
env : envmacros.vim: Catching LatexSuiteFileType event
comp : compiler.vim: Catching LatexSuiteFileType event
fold : folding.vim: catching LatexSuiteFileType
SyntaxFolds : Finished folding in   0.014520 seconds.
brak : brackets.vim: Catching LatexSuiteFileType event
view : texviewer.vim: Catching LatexSuiteFileType event
main : main.vim: Catching LatexSuiteFileType event
main : SetTeXOptions: sourcing maps
comp : +Tex_SetTeXCompilerTarget: setting target to [pdf] for Compiler
comp : +Tex_SetTeXCompilerTarget: setting target to [pdf] for Viewr
comp : compiler/tex.vim: sourcing this file
pack : packages.vim: Catching LatexSuiteFileType event
pack : +Tex_pack_updateall
pack : :Tex_pack_updateall: split
pack : +Tex_ScanForPackages
pack : :Tex_ScanForPackages: Begining scans in [animatedTransition.tex], beginline = 139
pack : :Tex_ScanForPackages: beginning scan for \usepackage lines
pack : :Tex_ScanForPackages: found package(s) ["hyperref"] on line 61
pack : :Tex_ScanForPackages: found package(s) ["graphicx"] on line 62
pack : :Tex_ScanForPackages: found package(s) ["times"] on line 63
pack : :Tex_ScanForPackages: found package(s) ["amsmath"] on line 64
pack : :Tex_ScanForPackages: found package(s) ["floatrow"] on line 66
pack : :Tex_ScanForPackages: found package(s) ["subfig"] on line 67
pack : :Tex_ScanForPackages: found package(s) ["booktabs"] on line 68
pack : :Tex_ScanForPackages: found package(s) ["paralist"] on line 69
pack : :Tex_ScanForPackages: found package(s) ["color"] on line 70
pack : :Tex_ScanForPackages: found package(s) ["enumitem"] on line 71
pack : :Tex_ScanForPackages: found package(s) ["multirow"] on line 72
pack : :Tex_ScanForPackages: found package(s) ["cite"] on line 80
pack : :Tex_ScanForPackages: found package(s) ["cite"] on line 83
pack : :Tex_ScanForPackages: End scan \usepackage, detected packages = ,hyperref,graphicx,times,amsmath,floatrow,subfig,booktabs,paralist,color,enumitem,multirow,cite,cite
pack : :Tex_ScanForPackages: Beginning scan for \newcommand's
pack : :Tex_ScanForPackages: Beginning scan for \newenvironment's
pack : -Tex_ScanForPackages
pack : :Tex_pack_updateall: detected [hyperref,graphicx,times,amsmath,floatrow,subfig,booktabs,paralist,color,enumitem,multirow,cite,cite] in first run
pack : :Tex_pack_updateall: scanning package hyperref
pack : :Tex_pack_updateall: findfile("hyperref.sty")
pack : :Tex_pack_updateall: did not find "hyperref.sty" in ".,"
pack : :Tex_pack_updateall: present file = animatedTransition.tex
pack : :Tex_pack_updateall: hyperref not found anywhere
pack : :Tex_pack_updateall: scanning package graphicx
pack : :Tex_pack_updateall: findfile("graphicx.sty")
pack : :Tex_pack_updateall: did not find "graphicx.sty" in ".,"
pack : :Tex_pack_updateall: present file = animatedTransition.tex
pack : :Tex_pack_updateall: graphicx not found anywhere
pack : :Tex_pack_updateall: scanning package times
pack : :Tex_pack_updateall: findfile("times.sty")
pack : :Tex_pack_updateall: did not find "times.sty" in ".,"
pack : :Tex_pack_updateall: present file = animatedTransition.tex
pack : :Tex_pack_updateall: times not found anywhere
pack : :Tex_pack_updateall: scanning package amsmath
pack : :Tex_pack_updateall: findfile("amsmath.sty")
pack : :Tex_pack_updateall: did not find "amsmath.sty" in ".,"
pack : :Tex_pack_updateall: present file = animatedTransition.tex
pack : :Tex_pack_updateall: amsmath not found anywhere
pack : :Tex_pack_updateall: scanning package floatrow
pack : :Tex_pack_updateall: findfile("floatrow.sty")
pack : :Tex_pack_updateall: did not find "floatrow.sty" in ".,"
pack : :Tex_pack_updateall: present file = animatedTransition.tex
pack : :Tex_pack_updateall: floatrow not found anywhere
pack : :Tex_pack_updateall: scanning package subfig
pack : :Tex_pack_updateall: findfile("subfig.sty")
pack : :Tex_pack_updateall: did not find "subfig.sty" in ".,"
pack : :Tex_pack_updateall: present file = animatedTransition.tex
pack : :Tex_pack_updateall: subfig not found anywhere
pack : :Tex_pack_updateall: scanning package booktabs
pack : :Tex_pack_updateall: findfile("booktabs.sty")
pack : :Tex_pack_updateall: did not find "booktabs.sty" in ".,"
pack : :Tex_pack_updateall: present file = animatedTransition.tex
pack : :Tex_pack_updateall: booktabs not found anywhere
pack : :Tex_pack_updateall: scanning package paralist
pack : :Tex_pack_updateall: findfile("paralist.sty")
pack : :Tex_pack_updateall: did not find "paralist.sty" in ".,"
pack : :Tex_pack_updateall: present file = animatedTransition.tex
pack : :Tex_pack_updateall: paralist not found anywhere
pack : :Tex_pack_updateall: scanning package color
pack : :Tex_pack_updateall: findfile("color.sty")
pack : :Tex_pack_updateall: did not find "color.sty" in ".,"
pack : :Tex_pack_updateall: present file = animatedTransition.tex
pack : :Tex_pack_updateall: color not found anywhere
pack : :Tex_pack_updateall: scanning package enumitem
pack : :Tex_pack_updateall: findfile("enumitem.sty")
pack : :Tex_pack_updateall: did not find "enumitem.sty" in ".,"
pack : :Tex_pack_updateall: present file = animatedTransition.tex
pack : :Tex_pack_updateall: enumitem not found anywhere
pack : :Tex_pack_updateall: scanning package multirow
pack : :Tex_pack_updateall: findfile("multirow.sty")
pack : :Tex_pack_updateall: did not find "multirow.sty" in ".,"
pack : :Tex_pack_updateall: present file = animatedTransition.tex
pack : :Tex_pack_updateall: multirow not found anywhere
pack : :Tex_pack_updateall: scanning package cite
pack : :Tex_pack_updateall: findfile("cite.sty")
pack : :Tex_pack_updateall: did not find "cite.sty" in ".,"
pack : :Tex_pack_updateall: present file = animatedTransition.tex
pack : :Tex_pack_updateall: cite not found anywhere
pack : :Tex_pack_updateall: scanning package cite
pack : :Tex_pack_updateall: cite already scanned
pack : Tex_pack_check: searching for hyperref in dictionaries/ in &rtp
pack : Tex_pack_check: searching for graphicx in dictionaries/ in &rtp
pack : Tex_pack_check: searching for times in dictionaries/ in &rtp
pack : Tex_pack_check: searching for amsmath in dictionaries/ in &rtp
pack : Tex_pack_check: searching for color in dictionaries/ in &rtp
pack : Tex_pack_check: searching for cite in dictionaries/ in &rtp
pack : Tex_pack_check: searching for cite in dictionaries/ in &rtp
pack : :Tex_pack_updateall: throwing LatexSuiteScannedPackages event
pack : -Tex_pack_updateall
comp : +Tex_RunLaTeX, b:fragmentFile = 0
comp : Tex_RunLaTeX: compiling to target [pdf]
comp : Tex_RunLaTeX: getting dependency chain = [pdf]
comp : +Tex_SetTeXCompilerTarget: setting target to [pdf] for Compiler
comp : Tex_RunLaTeX: setting target to pdf
comp : Tex_RunLaTeX: compiling file multiple times via Tex_CompileMultipleTimes
comp : Tex_CompileMultipleTimes: latex run number : 1
comp : Tex_CompileLatex: getting mainfname = [animatedTransition.tex] from Tex_GetMainFileName
comp : Tex_CompileLatex: execing [make! animatedTransition.tex]
comp : Tex_CompileMultipleTimes: errors = [
 1 /usr/local/texlive/2014/texmf-dist/tex/latex/caption/caption.sty warning: Package caption Warning: Unsupported document class (or package) detected, usage of the caption package is not recommended.]
comp : Tex_CompileMultipleTimes: Ran latex 1 time(s)
proj : texproject.vim: catching LatexSuiteFileType event
env : envmacros.vim: Catching LatexSuiteFileType event
comp : compiler.vim: Catching LatexSuiteFileType event
fold : folding.vim: catching LatexSuiteFileType
SyntaxFolds : Finished folding in   0.061558 seconds.
brak : brackets.vim: Catching LatexSuiteFileType event
view : texviewer.vim: Catching LatexSuiteFileType event
main : main.vim: Catching LatexSuiteFileType event
main : SetTeXOptions: sourcing maps
comp : +Tex_SetTeXCompilerTarget: setting target to [pdf] for Compiler
comp : +Tex_SetTeXCompilerTarget: setting target to [pdf] for Viewr
comp : compiler/tex.vim: sourcing this file
pack : packages.vim: Catching LatexSuiteFileType event
pack : +Tex_pack_updateall
pack : :Tex_pack_updateall: split
pack : +Tex_ScanForPackages
pack : :Tex_ScanForPackages: Begining scans in [/usr/local/texlive/2014/texmf-dist/tex/latex/caption/caption.sty], beginline = 0
pack : :Tex_ScanForPackages: beginning scan for \usepackage lines
pack : :Tex_ScanForPackages: End scan \usepackage, detected packages = 
pack : :Tex_ScanForPackages: Beginning scan for \newcommand's
pack : :Tex_ScanForPackages: Beginning scan for \newenvironment's
pack : -Tex_ScanForPackages
pack : :Tex_pack_updateall: detected [] in first run
pack : :Tex_pack_updateall: throwing LatexSuiteScannedPackages event
pack : -Tex_pack_updateall
comp : Tex_RunLaTeX: errlist = [
 1 /usr/local/texlive/2014/texmf-dist/tex/latex/caption/caption.sty warning: Package caption Warning: Unsupported document class (or package) detected, usage of the caption package is not recommended.]
comp : Tex_RunLaTeX: There were errors in compiling, breaking chain...
comp : Tex_SetupErrorWindow: mfnlog = caption.log
comp : not finding error pattern anywhere in quickfix window :
comp : -Tex_RunLaTeX`
